test/storage.test.js

```diff
@@ -15,6 +20,19 @@ describe('Tests for storage', () => {
     expect(result).toEqual(feed2);
   });
 
+  it('check feed count', async () => {
+    const result = await storage.getFeedsCount();
```

Comment:
Same issue here, and you're proving the problem: you depend on the previous test.  Each test in a test suite should be independent, and not depend on the order of test execution, or on other tests.